string mode
Local local
Path path
geometry_msgs/Point32 point